tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-integration-test.php

```
@@ -264,6 +276,207 @@ public function test_is_merchant_enabled_returns_false_when_disabled() {
+	 *
+	 * @return void
+	 */
+	public function test_is_feed_unchanged_returns_false_when_no_cached_upload() {
+		delete_option( $this->last_upload_option );
+
+		$integration  = new \WC_Stripe_Agentic_Commerce_Integration();
+		$is_unchanged = new \ReflectionMethod( \WC_Stripe_Agentic_Commerce_Integration::class, 'is_feed_unchanged' );
+		$is_unchanged->setAccessible( true );
+
+		$this->assertFalse( $is_unchanged->invoke( $integration, 'abc123' ) );
+	}
```

PHPMD (2.15.0)

[error] The method test_is_feed_unchanged_returns_false_when_no_cached_upload is not named in camelCase. (undefined)

(CamelCaseMethodName)